tests/libvmi/factory.go

```diff
@@ -40,20 +40,29 @@ func NewFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	return New(opts...)
 }
 
-// NewCirros instantiates a new CirrOS based VMI configuration
-func NewCirros(opts ...Option) *kvirtv1.VirtualMachineInstance {
+func newCirros(containerDisk string, opts ...Option) *kvirtv1.VirtualMachineInstance {
```

Comment:
This is no longer "Cirros", the name now is not representing what this function creates.
It is Cirros when the result is a VMI that is defined with a Cirros disk. But now the result is any disk.

I suggest you keep the duplication of options, these are two different VMI/s, the fact that they share common options is not enough to enforce a shared function.

Reply:
The function is private, and it is just a helper function to remove code duplication.

I can rename it but I think we should not have two functions alike like that.

Reply:
I removed it since I removed NewCirrosCustomLocation().